firestore/integration_test_internal/src/transaction_test.cc

```diff
@@ -91,38 +91,26 @@ class TransactionTest : public FirestoreIntegrationTest {
   }
 };
 
-class TestTransactionFunction : public TransactionFunction {
- public:
-  TestTransactionFunction(DocumentReference doc) : doc_(doc) {}
-
-  Error Apply(Transaction& transaction, std::string& error_message) override {
-    Error error = Error::kErrorUnknown;
-    DocumentSnapshot snapshot = transaction.Get(doc_, &error, &error_message);
-    EXPECT_EQ(Error::kErrorOk, error);
-    EXPECT_FALSE(snapshot.exists());
-    transaction.Set(doc_, MapFieldValue{{key_, FieldValue::String(value_)}});
-    return error;
-  }
-
-  std::string key() { return key_; }
-  std::string value() { return value_; }
-
- private:
-  DocumentReference doc_;
-  const std::string key_{"foo"};
-  const std::string value_{"bar"};
-};
-
 TEST_F(TransactionTest, TestGetNonexistentDocumentThenCreatePortableVersion) {
```

Comment:
The `PortableVersion` part can now be removed from the name of the test.

Reply:
Done